### ISSUE TITLE
[BUG] [aria-hidden="true"] elements contain focusable descendants

### ISSUE THREAD

**Issue by Sa7aR-Kassem** [TDRA-ae/aegov-dls#68]
Take a moment to review - is it really a bug? Or is it more of an enhancement or a typo in the documentation? A Bug is code that is broken, or when you try to implement a certain piece of code that is mentioned in the documentation, you have a result that is not the same as described.

Now if you are absolutely sure it is indeed a bug, please fill in the following template.

Mention what part of the design system the bug is related to
Is the bug part of a "Slider", 

More context in relation to the design system
If the bug is part of a component or a block, please mention which one. If the bug is related to the plugin, please mention if it is during installation, compiling or configuration.

Describe the bug
A clear and concise description of what the bug is. Help us understand what you are doing that is causing an error.

To Reproduce
Steps to reproduce the behaviour:

Use the chrome lighthouse tool
Analysis the page
[aria-hidden="true"] elements contain focusable descendants

A bit more context about the browser

Additional context

**Comment by moiz-conceptlz** [comment]
@Sa7aR-Kassem 

There is nothing that is part of the components that forces a `[aria-hidden="true"]` except for icons.

When you add an icon (like an image or an svg) you want to put `[aria-hidden="true"]` on it, but this also means that you cannot make it clickable. If you are going to add a <a> on an html attribute, you cannot add `[aria-hidden="true"]`

Hence, anything that is clickable will have focus. You cannot mark it as hidden. If you want to hide something on a mobile, and display it on a desktop -- which is a clickable element, then you must ideally also hide the parent container.

Please share a screenshot of which block or component is reporting this error with a screenshot.

**Comment by Sa7aR-Kassem** [comment]
I wanted to inform you that you have the same issue in the Hero component, as shown in the attached screenshot. This issue seems to be present in this specific component, and I thought it would be helpful to point it out.

Please let me know if you need any further details or if I can assist in resolving this matter.

![image](https://github.com/user-attachments/assets/e542161a-643a-4642-9aec-640fd9679eaf)

**Comment by moiz-conceptlz** [comment]
Hello @Sa7aR-Kassem 

This is great, thank you for pointing this out. Moving this back to a bug status and involving @anismansuriui to the conversation.

**Comment by Sa7aR-Kassem** [comment]
Hello @moiz-conceptlz @anismansuriui 

Do you have any recommendations for me?

**Comment by anismansuriui** [comment]
Hello @Sa7aR-Kassem 

Thank you for reporting this issue!

The behavior you’re observing is not related to any component of the plugin itself but rather to the implementation on the live DLS website. Specifically, it pertains to the responsive design where the sidebar is intentionally hidden (using aria-hidden="true") by default on small devices for the block pages.

To confirm, please try adding the slider or any other component from the plugin into your project and test it using Lighthouse. You’ll notice that this specific issue will not appear there.

If you encounter any issues directly related to the plugin, feel free to share more details, and I’ll be happy to assist!
